code/game/objects/items/devices/handtv.dm

```diff
@@ -40,6 +40,11 @@ var/global/list/camera_bugs = list()
 		user.unset_machine()
 		return
 
+/obj/item/device/handtv/attackby(obj/item/I, mob/user)
```

Comment:
This is very awkward, and it's different from how all of the other ways of taking pictures remotely work. What is the reason for not letting one click on the tile they want to take a photo of?

Reply:
1. To take a photo of a specific tile with a camerabug, you'd have to quickly switch from an activated handtv to a camera before the camera vision was gone
2. That method only works when the camera bug's sprite is still visible, like on walls.

Reply:
But do we agree that it would be better if none of that were the case?
You can make it so that as long as you have it *any* hand, switching hands wouldn't reset your view. 
You can address the second point by using `get_turf(the_camera_bug)` and calling get_hear on that turf rather than on the item the bug is attached to.

Reply:
I'll give that a try